### ISSUE TITLE
ZombieDriver::wait() does not honor the DriverInterface spec

### ISSUE THREAD

**Issue by 1ma** [minkphp/MinkZombieDriver#147]
It returns the result of a [```json_decode``` operation](https://github.com/minkphp/MinkZombieDriver/blob/master/src/ZombieDriver.php#L825), whereas Mink's DriverInterface specifies that this method should return a [boolean](https://github.com/minkphp/Mink/blob/master/src/Driver/DriverInterface.php#L597).

This prevents from using the following testing construct when using this driver:

```php
// Passes if the JS expression returns a truthy value before the timeout expires
$this->assertTrue($minkSession->wait(1000, "JS expression"));
```

**Comment by aik099** [comment]
Same/similar problem with other drivers?

**Comment by aik099** [comment]
I see no problem in changing https://github.com/minkphp/MinkZombieDriver/blob/master/src/ZombieDriver.php#L825 line to the

```php
return (bool)json_decode($this->server->evalJS($js));
```
as long as test passes. Would you please submit a PR?

**Comment by 1ma** [comment]
Sure.

**Comment by aik099** [comment]
Maybe, but this doesn't sound too critical a test (in main Mink repo) needs to be added/updated to ensure that `wait` method returns `true` and `false` specifically.

**Comment by stof** [comment]
Yeah, this may be buggy in case the JS expression does not return a real boolean

**Comment by stof** [comment]
The testsuite passes currently, because all expressions used in it are returning a real boolean

**Comment by aik099** [comment]
Closed via 39f6308923abf80d86114b2f158ac83dd77a89d4.
